src/bp/ui-studio/src/web/translations/en.json

```diff
@@ -23,6 +23,736 @@
     "supportedLanguages": "Supported languages",
     "website": "Website"
   },
+  "isoLangs": {
```

Comment:
I don't see the plus value of adding the translations of all known languages to man when we only support 2 for the UI and around 10 for the language server.  

It will be a pain to maintain when adding other languages (have fun translating those to russian!), and it will add to the bundle size.

Also, the list of language should be in the shared translation file (since admin and studio will use it).

Reply:
this is for the translations so I'm assuming that it would come from the language server and I think that clients can use their own language servers, no? if so we'd need to be able to support any language, but only in the languages available in the UI i.e. english/french for now, no?

Reply:
When we get to that, i'm sure there's something else we can use instead of having to duplicate those language key for every language we support on the UI, I just want to avoid having to maintain that list. And I definitely wouldn't put it in our "standard" translation file